### ISSUE TITLE
Cannot download 'Shooting Star' or any song from Spotify

### ISSUE THREAD

**Issue by AmazingGabriel16** [AllToMP3/alltomp3-app#152]
My operating system is: Windows 10
My AllToMP3 version is: 0.3.12
Song Link: https://open.spotify.com/track/7k5EcszNwUturoU20TTy1G?si=3-ino18MSZ25oGZPdJObyg

Problem, the song from the link above will not download on AllToMP3. A screenshot has been provided of the outcome when I do try to download it.

Image: 
![image](https://user-images.githubusercontent.com/23047921/76090060-fd145c00-600e-11ea-9a66-200c71ddd663.png)

**Comment by shahzebgit** [comment]
YouTube and Sound Cloud are working until then try those source Spotify is Stuck again, might get fixed _Hopefully :D_

**Comment by AmazingGabriel16** [comment]
"try those source Spotify is Stuck again"

What do you mean by this? I don't get it.

**Comment by UdaraAlwis** [comment]
Seems like Spotify has identified this loophole and blocked it. :( That's why AllToMP3 is no longer working. It's not legal to scraping off MP3 songs from Spotify, they've blocked it from all kinds of aspects with a complete blackbox of their service. It was some incredible work from the developer of this tool to figure out a loophole to override their blackbox.
Hopefully someone figure out a new loophole and the developer would be able to fix this tool once again. :) Cheers all!

**Comment by nuganics** [comment]
I'm using Sidify and it is working.

**Comment by AmazingGabriel16** [comment]
Interesting...

**Comment by nuganics** [comment]
A .dll change allows you to go over 3 minutes in Sidify. Sidify just plays and records so might not be as good as AlltoMp3?

**Comment by AmazingGabriel16** [comment]
So essentially, its also like Spytify.

OOF!!!

Ok, so to conclude, AllToMP3 is the only one that actually downloads and not records XD

I do believe though that with AllToMP3, what it does is that it searches for the content on YouTube, as before I tried downloading the Queen of Mars by Phineas and Ferb and it gave me the recording of a TV sound XD

**Comment by 616659** [comment]
Then until the dev finds another way to get mp3 downloads from Spotify, I should be using that.. Sidify thing? This is quite sad

**Comment by nuganics** [comment]
yeah I can send you the dll for Sidify or try https://jwallet.github.io/spy-spotify/overview.html

**Comment by 616659** [comment]
@nuganics thx, but I am a mac user.. sad..

**Comment by AmazingGabriel16** [comment]
> 
> 
> @nuganics thx, but I am a mac user.. sad..

You can use [OBS](https://obsproject.com/).

As Sidify is simply a recorder, you can do the same with OBS which is compatible with mac also.

Hope this helps!

**Comment by nuganics** [comment]
Can also use Audacity for Mac also just a bit of a manual effort.

**Comment by NTag** [comment]
The latest update (0.3.17) should fix that!

**Comment by 616659** [comment]
Wait spotify download works now?
